docs/usage/quickstart.rst

```diff
@@ -3,85 +3,119 @@
    Quickstart
    subdirectory
    Todo
-   usernames
 
 Quickstart Guide
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines running
-CentOS_ 7.4 to which you have SSH access (these can be VMs). Each machine
-acting as a Kubernetes_ node (all of them, in this example) also need to have at
-least one disk available to provision storage volumes.
+
+To set up a testing cluster quickly using MetalK8s_ requires three machines
+running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
+machine acting as a Kubernetes_ node (all of them, in this example) must also
+have at least one disk available to provision storage volumes.
 
 .. todo:: Give some sizing examples
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-Defining an Inventory
----------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+Clone or Copy the MetalK8s Git Repo
+-----------------------------------
+
+Clone the MetalK8s project from GitHub:
+
+.. code-block:: shell
+
+   $ git clone https://github.com/scality/metal-k8s
```

Comment:
Since now we have released versions, do we really want to tell people to use whatever is latest in Git?

Reply:
Good question. We should give instructions to get the latest 'stable' version. Which one is that?

Reply:
Agree @rahulreddy, though I wouldn't go too in-depth on Git.